lib/rubocop/cop/style/redundant_parentheses.rb

```diff
@@ -126,8 +128,17 @@ def only_closing_paren_before_comma?(node)
           line_range.source =~ /^\s*\)\s*,/
         end
 
-        def disallowed_literal?(node)
-          node.literal? && !ALLOWED_LITERALS.include?(node.type)
+        def disallowed_literal?(begin_node, node)
```

Comment:
I've used the same convention as in other places in the file. See check, check_unary, call_chain_starts_with_int? methods.

That was wrong?